src/components/ShortcutTile.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { useClient, useFetchShortcut } from 'cozy-client'
+import get from 'lodash/get'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+
+const ShortcutTile = ({ file }) => {
+  const client = useClient()
+  const { shortcutInfos } = useFetchShortcut(client, file._id)
+  const url = get(shortcutInfos, 'data.attributes.url', '#')
+  const shortcurtIcon = get(shortcutInfos, 'data.attributes.metadata.icon')
+  const label = file.name.substr(0, file.name.lastIndexOf('.'))
+  const { isMobile } = useBreakpoints()
+
+  return (
+    <a href={url} className="item">
+      <div className="item-icon">
+        {shortcurtIcon ? (
+          <img
+            src={`data:image/svg+xml;base64,${btoa(shortcurtIcon)}`}
+            width="100%"
```

Comment:
no 32 / 40 size for the image ?

Reply:
Nah with `100%` it auto-adjusts it's size.